### DISCUSSION TITLE
MySQL extension: Support for encrypted connections?

### DISCUSSION THREAD

**Discussion by renzepost** [duckdb/duckdb#12676] [Ideas]
I'm trying to connect to a MySQL database that only allows [encrypted connections](https://dev.mysql.com/doc/refman/8.4/en/using-encrypted-connections.html), but there doesn't seem to be a way to do that in DuckDB.
I have a custom CA certificate file that I can normally use in the MySQL client (the `ssl_ca` parameter). DuckDB has no such parameter so a connection can't be established. Unless I'm overlooking something, it seems DuckDB doesn't support encrypted connections to MySQL. Any help would be greatly appreciated!

**Comment by aszenz** [comment]
I'm facing the same issue, our db is public facing and is therefor using encryption

**Comment by aszenz** [comment]
Created issue https://github.com/duckdb/duckdb_mysql/issues/71
